test/integration/consul-container/libs/utils/helpers.go

```diff
@@ -4,7 +4,16 @@
 package utils
 
 import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+
 	"github.com/hashicorp/consul/api"
+	"gopkg.in/square/go-jose.v2"
+	"gopkg.in/square/go-jose.v2/jwt"
```

Comment:
https://github.com/go-jose/go-jose is the latest v3 of this package

Reply:
looks like we are still using go-jsose v2 everywhere else, so I wonder if it is worth using v3 here